AndroidXCI/lib/src/test/kotlin/dev/androidx/ci/gcloud/GoogleCloudApiPlaygroundTest.kt

```diff
@@ -66,6 +65,27 @@ internal class GoogleCloudApiPlaygroundTest {
         )
     }
 
+    @Test
+    fun copyItem() = testScope.runTest {
```

Comment:
note that these tests do not run in CI as they use prod (just wanted to make sure you know it).

for testing, we should have all the fakes you need here (i assume)

Reply:
yes. will be adding actual tests.